xml/ssh_banners.xml

```diff
@@ -1732,6 +1732,7 @@
 
   <fingerprint pattern="^SSH Protocol Compatible Server SCS (.*)$">
     <description>Netscreen with version</description>
+    <example service.version="2.0">SSH Protocol Compatible Server SCS 2.0</example>
```

Comment:
I don't see this value in any of our datasets (I searched SSH and FTP data) but it has been documented on the internet and matches regex used in Nmap.

https://www.chiark.greenend.org.uk/~sgtatham/putty/wishlist/netscreen-command.html
https://rt.cpan.org/Public/Bug/Display.html?id=6256

I wonder if this was observed on an SFTP service or if it should have been in the SSH fingerprints file. Either way, it seems to be of little consequence.

Reply:
https://search.censys.io/search?resource=hosts&sort=RELEVANCE&per_page=25&virtual_hosts=EXCLUDE&q=services.ssh.endpoint_id.raw%3A%22SSH+Protocol+Compatible+Server+SCS+2.0%22

Reply:
Thanks. yeah, we should make sure that we get a fingerprint for this in the SSH banners file.

Reply:
Not following -- this is ssh_banners.xml?